src/server/network/protocol/protocolgame.cpp

```diff
@@ -1719,7 +1719,7 @@ void ProtocolGame::parseSetOutfit(NetworkMessage &msg) {
 				g_logger().debug("Bool isMounted: {}", isMounted);
 			}
 
-			uint8_t isMountRandomized = msg.getByte();
+			uint8_t isMountRandomized = !oldProtocol ? msg.getByte() : 0;
```

Comment:
v8 old protocol no send "isMountRandomized" , las packet  is newOutfit.lookMount = msg.get<uint16_t>(); (L1711)

![image](https://github.com/user-attachments/assets/b7043095-f016-464d-926d-e0597f2ac329)
https://github.com/opentibiabr/otcv8/blob/main/src/client/protocolgamesend.cpp#L833

Redemption 11.00 no send isMountRandomized only in 13.40